src/plan/global.rs

```diff
@@ -299,6 +299,16 @@ pub trait Plan: 'static + HasSpaces + Sync + Downcast {
         true
     }
 
+    /// Return whether the current GC may move any object.  The VM binding can make use of this
+    /// information and choose to or not to update some data structures that record the addresses
+    /// of objects.
+    ///
+    /// This function is callable during a GC.  From the VM binding's point of view, the information
+    /// of whether the current GC is a defrag GC is available since `Collection::stop_mutators` is
```

Comment:
Mentioning defrag could be confusing here.
```suggestion
    /// of whether the current GC moves object or not is available since `Collection::stop_mutators` is
```